### DISCUSSION TITLE
Where are sway and waybar variables defined?

### DISCUSSION THREAD

**Discussion by chilledtonic** [Manjaro-Sway/manjaro-sway#676] [Q&A] [answered]
In the standard waybar config, it has many variables defined that I can't find the source of.

For example, under the "backlight" widget, it shows $brightness_up & $brightness_down as the two scripts it calls on scroll; this keeps the config clean but since I don't know where $brightness_up is defined I can't modify it, or any others.

Any ideas?

**Comment by boredland** [comment, marked as answer]
All our definitions are in `/etc/sway`. You can also browse and search it through GitHub: https://github.com/manjaro-sway/desktop-settings/tree/sway/community/sway/etc/sway

**Reply by boredland** [reply]
So: the variables are regular sway variables. You should be able to override them via your user space `.config/sway/definitions.d`
